Library/Formula/ansible.rb

```diff
@@ -331,11 +337,6 @@ class Ansible < Formula
     sha256 "b00a70ba72b068adb6aaca19b14ed4bde9c950ef4f24234d095f9f9985073179"
   end
 
-  resource "pytz" do
-    url "https://pypi.python.org/packages/source/p/pytz/pytz-2015.7.tar.bz2"
-    sha256 "fbd26746772c24cb93c8b97cbdad5cb9e46c86bbdb1b9d8a743ee00e2fb1fc5d"
-  end
-
```

Comment:
I'm not sure which hunk you'd like me to remove.  The patch moves pytz to a different location in the file, the first hunk inserts it into the new location and the second hunk deletes it from the old.